app/views/profile/orders/show.html.erb

```diff
@@ -16,6 +16,47 @@
 </section>
 <p>Total number of items: <%= @order.item_count %></p>
 <p>Grand total: <%= number_to_currency(@order.grandtotal) %></p>
-<% if @order.status != 'shipped' %>
-  <%= link_to 'Cancel Order', "/profile/orders/#{@order.id}" %>
-<% end %>
```

Comment:
Did this functionality get moved to another location? For one of the user stories I completed it's necessary to have a Cancel Order link included with each order. I didn't see this anywhere else, so I'm concerned it could break tests.

Reply:
Good catch!! Swear i kept this during a merge conflict but apparently not